app/assets/stylesheets/modules/_course.styl

```diff
@@ -199,6 +199,7 @@ rating(color)
 
     .course-details
       width 90%
+      max-width 920px
```

Comment:
This section is part of the media query, and only applies below 920px screen width. The max-width should be in the same class as the changed width above.

Reply:
I'll change it